### PR TITLE
Documentation: Add new quickstart + remove cleanup repo

### PR DESCRIPTION
**Pull request by Willmish**
Issue Number: (Link to Github Issue or Azure Dev Ops Task/Story)

## Summary
Repository cleanup as per #3 and new documentation for getting started + examples for the CLI, deploying Web API locally, calling the Web API with HTTP requests, generating API clients and example of using a generated Python client.

## Changes

- Removes `carbon-aware-sdk` directory along with its content
- Modify [docs/quickstart.md](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/quickstart.md)
- Add reference to the quickstart guide in the [GettingStarted.md](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/GettingStarted.md) guide

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [x] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [x] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?
No

## Is this a breaking change?
No - documentation change

## Anything else?
Other comments, collaborators, etc.
This PR Closes Issue #3 
Works towards #22 and #78

### CODE REVIEW

**Comment by Willmish** [comment]
Please take a look at it @danuw as well!

File: carbon-aware-sdk/license.md [hunk 1]
```diff
@@ -1,39 +0,0 @@
-Materials in this repository other than source code are provided as follows:
```
Comment (bderusha) [inline carbon-aware-sdk/license.md:1]: @Willmish Did you mean to delete this license?  Either way it is linked in the [readme.md](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/README.md?plain=1#L43) file which should be updated.

Comment (danuw) [reply carbon-aware-sdk/license.md:1]: Hi @bderusha, I have seen your PR at https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/105. 
We should prabably move the license following the guidance from #105.
Would we move it here or in #105  please? If you are happy, let's get that merge and update 105 to have the license in the place.
(Will that also help feed into this documentation better? https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/41 - I am not I understand where the guidance is on how to ensure that online documentation gets updated inline with the repo)

**Review by vaughanknight — APPROVED** [review]
Approved.

**Pull request merged**